builders/xcm/xtokens.md

```diff
@@ -0,0 +1,248 @@
+
+Once the transaction is processed, the **TargetAccount** should have received the transferred amount minus a small fee that is deducted to execute the XCM on the destination chain. On Polkadot.js Apps, can check the relevant extrinsics and events in [Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer/query/0xf163f304b939bc10b6d6abcd9fd12ea00b6f6cd3f12bb2a32b759b56d2f1a40d) and the [relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/explorer/query/0x5b997e806303302007c6829ab8e5b166a8aafc6a68f10950cc5aa8c6981ea605)
+
+### X-Tokens Transfer MultiAsset Function {: #xtokens-transfer-multiasset-function}
+
+In this example, you'll build an XCM message to transfer `xcUNIT` from Moonbase Alpha back to its [relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts). 
+
+To transfer assets using the `transferMultiasset` function, first head to the extrinsic page of [Polkadot JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/extrinsics). To be able to send an extrinsic, you need to have an [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts) with [funds](https://docs.moonbeam.network/builders/get-started/networks/moonbase/#get-tokens) first. Also, you'll need `xcUNIT`
+
+
+Once you have everything set up, set the following options:
+
+1. Select the account from which you want to send the XCM
+2. Choose the **xTokens** pallet
+3. Choose the **transferMultiasset** extrinsic
+4. To define the XCM asset multilocation, you have to target `UNIT` in the relay chain from Moonbase Alpha as the origin. Each chain sees its own asset differently. Therefore, you will have to set a different asset multilocation for each destination. For this example, the relay chain token can be defined as:
+
+    | Parameter | Value  |
+    |:---------:|:------:|
+    |  Version  |  *V1*  |
+    |  Parents  |  *1*   |
+    | Interior  | *Here* |
+
+5. Set the asset type as **Fungible**
+6. Set the number of tokens to send. For this example, you are sending 1 `xcUNIT`, but you have to account for the 12 decimals
+7. To define the XCM destination multilocation, you have to target an account in the relay chain from Moonbase Alpha as the origin. Therefore, set the following parameters:
+
+    | Parameter |      Value      |
+    |:---------:|:---------------:|
+    |  Version  |      *V1*       |
+    |  Parents  |       *1*       |
+    | Interior  |      *X1*       |
+    |    X1     |  *AccountId32*  |
+    |  Network  |      *Any*      |
+    |    Id     | *TargetAccount* |
+
+8. Set the destination weight to `1000000000`. Note that on Moonbase Alpha, each XCM instruction costs around 100000000 weight units. A `transferMultiasset` consists of 4 XCM instructions, so a destination weight of 400000000 should be enough
```

Comment:
I would format the other weight numbers to the be the same format as with the initial `1000000000`

Reply:
Good point